src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,44 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int supplyAmount = 0;
+        int buyAmount = 0;
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                String[] parts = line.split(",");
+                if (line.startsWith("supply")) {
+                    supplyAmount += Integer.parseInt(parts[1]);
+                } else {
+                    buyAmount += Integer.parseInt(parts[1]);
+                }
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from the file " + fromFileName, e);
+        }
```

Comment:
encapsulate this logic into private methods - one to read raw data as is, and a second one to parse it, we should [separate our concerns](https://en.wikipedia.org/wiki/Separation_of_concerns#:~:text=In%20computer%20science%2C%20separation%20of,code%20of%20a%20computer%20program.)

here (since business logic isn't super trivail), public method should only contain a few calls to the private methods, that do the actual work, more or less like this

    String data = readFile(fromFileName);
    String report = generateReport(data);
    writeToFile(toFileName, report);